### PR TITLE
Zynqmp 2018 3

### PR DESCRIPTION
**Pull request by chrisguikema**
Test with: https://github.com/seL4/camkes-vm-examples/pull/57

### CODE REVIEW

**Review by axel-h — APPROVED** [review]
Looks ok to me. Should this also have a CI run then?

**Comment by chrisguikema** [comment]
> Should this also have a CI run then?

My thought is to make an update to the ci-actions repo to add a target for each Petalinux version

**Comment by axel-h** [comment]
> My thought is to make an update to the ci-actions repo to add a target for each Petalinux version

With the fix from https://github.com/seL4/ci-actions/pull/316 passing that as parameter should work then.

File: components/VM_Arm/CMakeLists.txt [hunk 1]
```diff
@@ -140,7 +140,9 @@ config_option(
 config_choice(
     VmZynqmpPetalinuxVersion
     ZYNQMP_PETALINUX_VERSION
-    "The version of Petalinux to use"
+    "The version of Petalinux to use. Default is 2018.3, which will images compatible \
+    with the seL4 device tree"
```
Comment (lsf37) [inline components/VM_Arm/CMakeLists.txt:144]: Something missing in the sentence. Did you mean "use"?

```suggestion
    "The version of Petalinux to use. Default is 2018.3, which will use images compatible \
    with the seL4 device tree"
```

Comment (chrisguikema) [reply components/VM_Arm/CMakeLists.txt]: Yup, definitely missed a word there

**Review by lsf37 — APPROVED** [review]

**Comment by axel-h** [comment]
Seems good to merge. We see the test failures because https://github.com/seL4/camkes-vm-examples/pull/57 need to be rebased to master to pick up the fixed for `vm_introspect` from https://github.com/seL4/camkes-vm-examples/pull/39

**Comment by axel-h** [comment]
I've rebased https://github.com/seL4/camkes-vm-examples/pull/57 and the basline builds pass now. So we could merge this PR now to have a dependency of the other PRs in. What remains is making the ping examples work on the ZCU102. The run still fails, see https://github.com/seL4/camkes-vm/pull/124

**Review by lsf37 — APPROVED** [review]
Happy with merging this part now.

**Pull request merged**